cpp/arcticdb/entity/types.hpp

```diff
@@ -117,7 +117,7 @@ enum class ValueType : uint8_t {
     ASCII_DYNAMIC = 12,
     /// Used to represent null types. Each type can be converted to Empty and Empty can be converted to each type.
     EMPTY = 13,
-    COUNT // Not a real value type, should not be added to proto descriptor. Used to count the number of items in the enum
+    COUNT = 14 // Not a real value type, should not be added to proto descriptor. Used to count the number of items in the enum
```

Comment:
Adding specific value will force people to change it every time when adding new types and potentially lead to bugs if they forget. I don't see any particular benefit of this change.

Reply:
It is just that clang-tidy was complaining about it, and I would like as much as possible for it to be clean